Automattic/wp-calypso
# Woo-Connect Insights: update price to quantity and added currency support

This PR intends to introduce field format types so that values are appropriately formatted in the widgets and lists. A new `formatValue()` util has been added with tests. it accepts 'text' (string), 'number' (formatted to 2 decimals) and currency (formatted by `formatCurrency()` without code for now).

Fixes #15763 

To test:
- `ENABLE_FEATURES=woocommerce/extension-stats make run`
- browse to `/store/stats/orders/day/{ slug }` of a site with WooCommerce and Jetpack syncing
- check the 3 lists at the bottom of the page to ensure that the names, quanity and sales are formatted as you'd expect

There is another PR out to introduce the currency codes here - #15846
